app/App/Account/Account/Signer/SignerPreview/index.js

```diff
@@ -36,7 +36,11 @@ class Signer extends React.Component {
     if (this.resizeObserver) this.resizeObserver.disconnect()
   }
 
-  verifyAddress () {
+  verifyAddress (signerType) {
+    if (['ledger', 'trezor'].includes(signerType)) {
```

Comment:
I think we have an `isHardwareSigner()` function somewhere. should we use this instead to cover all cases?